### PR TITLE
Fix accidentally added item in composer

### PR DESCRIPTION
**Pull request by reyostallenberg**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | -
| Related issues/PRs | -
| License | MIT

#### What's in this PR?
A fix of an accidentally added item in composer.json

#### Why?
Makes composer.json valid

#### Example Usage
-

#### BC Breaks/Deprecations
-

Describe BC breaks/deprecations here. (remove this section if not needed)
- 

#### To Do
-

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,7 +15,6 @@
         "symfony/translation": "^3.4 || ^4.0",
         "twig/twig": "^1.26 || ^2.0"
     },
-    "excelwebzone/recaptcha-bundle": "^1.4.2",
```
Comment (alexander-schranz) [inline composer.json]: This should be a test dependency.

Comment (reyostallenberg) [reply composer.json]: Okey, fixed that

Comment (alexander-schranz) [reply composer.json]: Thank you!

**Comment by alexander-schranz** [comment]
@reyostallenberg Thank you!

**Pull request merged**